document/core/exec/instructions.rst

```diff
@@ -2625,8 +2625,21 @@ Control Instructions
 :math:`\TRY~\blocktype~\instr_1^\ast~(\CATCH~x~\instr_2^\ast)^\ast~(\CATCHALL~\instr_3^\ast)^?~\END`
 ....................................................................................................
 
-.. todo::
-   Add prose for the |TRY| - |CATCH| - |CATCHALL| execution step.
+1. Assert: due to :ref:`validation <valid-blocktype>`, :math:`\expand_F(\blocktype)` is defined.
+
+2. Let :math:`[t_1^n] \to [t_2^m]` be the :ref:`function type <syntax-functype>` :math:`\expand_F(\blocktype)`.
+
+3. Assert: due to :ref:`validation <valid-try-catch>`, there are at least :math:`n` values on the top of the stack.
+
+4. Pop the values :math:`\val^n` from the stack.
+
+5. Let :math:`L` be the label whose arity is :math:`m` and whose continuation is the end of the |TRY| instruction.
+
+6. Let :math:`H` be the :ref:`catching exception handler <syntax-handler>` :math:`\CATCHadm\{a_x~instr_2^\ast\}^\ast\{\epsilon~\instr_3^\ast\}^?`, whose mapping from tag addresses to branch targets corresponds to the |CATCH| and |CATCHALL| clauses, where :math:`a_x` is the tag address of each tag :math:`x`.
+
+7. :ref:`Enter <exec-instr-seq-enter>` the block :math:`H~(\val^n~\instr_1^\ast)~\END` with label :math:`L`.
+
+8. :ref:`Enter <exec-handler-enter>` the exception handler `H`.
```

Comment:
I think I understand 7, but I was confused by 8. What does "entering the exception handling `H`" mean? Does that mean the exception is caught? (If so, can we assume it is caught?)

Reply:
I was in fact describing step 8, so the steps in my previous comment are the meaning of "entering the exception handler `H`", that is by putting the `H` (either a `catch[a? instr2*}*` or a `delegate{l}`) on the stack and starting to execute the instructions `instr*` of the try block. 

I now realised we also call this "installing an exception handler" (see [note here](https://webassembly.github.io/exception-handling/core/exec/instructions.html#entering-an-exception-handler-h)), which is perhaps a better description of what's happening, rather than "entering an exception handler". Switching to that.

Reply:
Done. @aheejin WDYT?

Reply:
Done. @aheejin WDYT?

Reply:
After [this suggestion](https://github.com/WebAssembly/exception-handling/pull/224#discussion_r972825235) this is now switched to the formulation "Enter instr* with exception handler H".